.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
do you have instructions for rebasing? I tried to do it this morning with yesterday's PR and could not figure out how to properly squash the file from the commit.